code/datums/shuttles.dm

```diff
@@ -21,7 +21,7 @@
 	mappath = "[prefix][shuttle_id].dmm"
 	. = ..()
 
-/datum/map_template/shuttle/load(turf/T, centered)
+/datum/map_template/shuttle/load(turf/T, centered, register=TRUE)
```

Comment:
This is necessary because preview shuttles loaded by the manipulator need to not override the existing escape shuttle until it's fully loaded.